controller/replica_controller.go

```diff
@@ -240,9 +240,9 @@ func (rc *ReplicaController) syncReplica(key string) (err error) {
 			return errors.Wrapf(err, "failed to cleanup the related replica instance before deleting replica %v", replica.Name)
 		}
 
-		rs, err := rc.ds.ListReplicasByNodeRO(replica.Spec.NodeID)
+		rs, err := rc.ds.ListVolumeReplicas(replica.Spec.VolumeName)
```

Comment:
I think the change here is fine in general, but it looks like we can use `ListVolumeReplicasRO` for efficiency.

Reply:
Good idea! Thank you. Fixed